app/form_models/review_assessment_details_form.rb

```diff
@@ -11,6 +11,12 @@ class ReviewAssessmentDetailsForm
     consultation_summary
     additional_evidence
     publicity_summary
+    amenity
+  ].freeze
+
+  PRIOR_APPROVAL_ASSESSMENT_DETAILS = %i[
```

Comment:
Is it not worth just explicitly saying what assessment details are necessary for both ldc and prior approval. Just thinking when we add more application types it might get a bit messy subtracting these from eachother.

i.e. 
```
PRIOR_APPROVAL_ASSESSMENT_DETAILS = %i[
    summary_of_work
    site_description
    consultation_summary
    additional_evidence
    publicity_summary
    amenity
  ].freeze

LDC_ASSESSMENT_DETAILS = %i[
    summary_of_work
    site_description
    consultation_summary
    additional_evidence
  ].freeze
  ```
  
  Then the validation method can be a bit easier to read and add onto when we get further application types.
  
  Another idea might be to add a has_and_belongs_to_many between application type and this assessment detail table (maybe one for householder though)

Reply:
I've done half your suggestion, not sure I'll have time to do the other half today